Quantori.AllureTestOpsClient/Model/GetTestCasesResponse.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+
+namespace Quantori.AllureTestOpsClient.Model
```

Comment:
In general, we create separate files for separate classes. Any particular reason to combine them in one file?

Reply:
It was auto-generated by swagger -> class converted. I am going to move the inner classes to separate files later.